nodejs/week1/avg.js

```diff
@@ -0,0 +1,17 @@
+HYF_Lesson_2_status.sql//process.argv is an array containing the command line arguments.
+const getCmdArgs = process.argv.slice(2);
```

Comment:
Small suggestion: for better understanding of code, it is better to use meaningful names. In my perception the variable name `getCmdArgs` sounds like a method name. Accordingly, I would suggest the name `commandArguments` or `cmdArgs`  or simply `numbers`

Reply:
sure i will update my code as per suggestion. Thank you.